Source/WebKit/Shared/WebPreferencesDefaultValues.cpp

```diff
@@ -204,12 +204,17 @@ bool defaultMediaSessionCoordinatorEnabled()
 
 bool defaultRunningBoardThrottlingEnabled()
```

Comment:
This seems to indicate that RunningBoard throttling is only enabled on macOS. Surely we do RunningBoard throttling on iOS too, no?